tests/container/vnc-script.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# vnc-script
+#
+# Copyright (C) 2016 Intel Corporation
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+set -e
+set -x
+# this runs inside the container. Returns 0 on success.
+ls -al /workdir
+# make a vncuser from the skeleton
+useradd -m --skel=/etc/vncskel vncUser
+
+# install xdpyinfo in the distros that are missing it
+if [ -x "$(command -v dnf)" ]; then
+    dnf -y install xorg-x11-utils
+else
+    if [ -x "$(command -v zypper)" ]; then
+        zypper --non-interactive install xdpyinfo
+
+    else
+        if [ -x "$(command -v yum)" ]; then
+            yum -y install xorg-x11-utils
+        fi
+    fi
+fi
+
+exec sudo -H -u vncUser /bin/bash - <<EOF
+
+# start the server
+vncserver -rfbport 5900 -name YOCTO :1
+sleep 2
+
+if grep Listening  /home/vncUser/.vnc/*.log | grep 5900 ; then
+echo "port established"
+else
+echo "port NOT established"
+exit 1
+fi
+DISPLAY=:1 xdpyinfo | grep "number of screens"
```

Comment:
this tests that the vnc server is up and a desktop is available.  If i do the more complicated image compare, I'll do it from the poky level where I have a --vnc flag added to autostartup the vncserver. since it needs a test too, I'm separating it that way.